### ISSUE TITLE
Nuxt parsing MD files??? invalid??

### ISSUE THREAD

**Issue by livelyPeng** [nuxt/nuxtjs.org#638]
build: {
    cssSourceMap: false,
    extractCSS:true,
    vendor: ['axios'],
    transpile: [/^element-ui/],
    loader:[
      {
         test:/\.sass$/,
         loader:'style-loader!css-loader!sass-loader',
      },
    //  invalid  invalid invalid invalid invalid invalid invalid
      {
        test: /\.md$/,
        use: [
          { loader: 'html-loader' },
          { loader: 'markdown-loader', options: {} }
        ]
      }
    ],
    styleResources:{
      scss:'./themes/common.scss'
    },
    extend(config, ctx) {
    }
  }

**Comment by manniL** [comment]
Hey 👋🏻

Unfortunately I don't understand the problem.
Could you rephrase your issue and ideally follow the issue template?

**Comment by debs-obrien** [comment]
closing this due to inactivity.
